packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -2327,6 +2329,17 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     }
   }
 
+  int get editableTextId => _editableTextId;
+  int _editableTextId;
+  set editableTextId(int value) {
+    assert(value != null);
+    if (value != _editableTextId) {
+      _editableTextId = value;
+      markNeedsPaint();
```

Comment:
Why does this require a paint update (seems to be only used in describeSemanticsConfiguration)?

Reply:
Done. For some reason I thought a semantics change always implied a repaint, but I guess that doesn't have to be the case.